### ISSUE TITLE
How about a distributed autokeras that is compatible with Google CoLab?

### ISSUE THREAD

**Issue by shyamalschandra** [jhfjhfj1/autokeras#306]
How about a distributed autokeras that is compatible with Google CoLab?

### Feature Description
I would like to use the hopeful distributed autokeras with the GPU and TPU runtime without doing major surgery to the code.  If that is impossible in the near future, I can go ahead and do it myself.

### Reason
It would be useful to accelerate the computation.

### Solution
Make it happen.

### Alternative Solutions
Slow and serial search.

### Additional Context
The tdqm library doesn't work on Google CoLab.  I changed it to the pyprind for compatibility with the Google Colab.  You can check it out at the following URL: https://github.com/shyamalschandra/autokeras

Thank you!

**Comment by tl-yang** [comment]
Hi @shyamalschandra , thank you for the suggestion. There are two things I would like to discuss with you before we start.

1. You mentioned that tdqm doesn't work on Google CoLab and you use pyprind to address the issue. I would prefer to change from 
```python  
from tqdm.autonotebook import tqdm
``` 

to 

```python
from tqdm import tqdm
```  
which can also resolve the issue without having to change the dependency.
I'll open up another issue for this problem.

2. Could you please explain how you are going to make Autokeras run distributely? 
    Providing some code snippets will be very helpful. 

Thanks a lot!
